transposon/revise_annotation.py

```diff
@@ -481,8 +460,15 @@ def determine_seed_stop(self, seed_stop, array_of_hits):
             seed_stop (int): The value for the seed stop.
         """
         for hit_index in array_of_hits:
-            if self.search_frame.loc[hit_index,].Stop > seed_stop:
-                seed_stop = self.search_frame.loc[hit_index,].Stop
+            if (
+                self.search_frame.loc[
+                    hit_index,
+                ].Stop
+                > seed_stop
+            ):
+                seed_stop = self.search_frame.loc[
+                    hit_index,
+                ].Stop
```

Comment:
this formatting is quite bizarre

note that you are calling this twice `self.search_frame.loc[hit_index,].Stop`

I recommend storing the search_frame result , for a future commit:
```
new_stop = self.search_frame.loc[hit_index,].Stop
if new_stop > seed_stop:
    seed_stop = new_stop
```

or you can use a ternary operator, but that might be too long of a line,
but may be more difficult to read
```
seed_stop = new_stop if new_stop > seed_stop else seed_stop
```